src/chart/helper/pieHelper.ts

```diff
@@ -24,11 +24,12 @@ import { parsePercent } from 'zrender/src/contain/text';
 
 export function getSectorCornerRadius(
     model: Model<{ borderRadius?: string | number | (string | number)[] }>,
-    shape: Pick<Sector['shape'], 'r0' | 'r'>
+    shape: Pick<Sector['shape'], 'r0' | 'r'>,
+    zeroIfNull?: boolean
 ) {
     let cornerRadius = model.get('borderRadius');
     if (cornerRadius == null) {
-        return null;
+        return zeroIfNull ? {innerCornerRadius: 0, cornerRadius: 0} : null;
```

Comment:
I think it's fine to return `{ innerCornerRadius: 0, cornerRadius: 0 }` directly. It will make this function simpler.

Reply:
But that will make the other states incorrect. For example, we specify the `borderRadius` as `5` in the normal state, what expected should be also `5` in the emphasis state, but if we use directly `{ innerCornerRadius: 0, cornerRadius: 0 }`, it will break this. Thus using `null` is reasonable.

Reply:
You are right, I totally forgot that.